docs/README.md

```diff
@@ -1,6 +1,8 @@
-# Juniper documentation
+# Welcome
 
-Welcome to the docs!
+This website provides technical documentation for the Juniper codebase.
```

Comment:
I suggest we avoid usage of internal code names like "Juniper" for things that are public-facing or publicly available.

Reply:
@thekaveman when we initially discussed the ticket for setting up docs, @hunterowens had also resolved it was time to rename this repo to juniper (#84) so I aligned towards that